src/main/webapp/components/results/results.js

```diff
@@ -322,7 +354,14 @@ const Container = props => {
     { error, loading, refetch },
```

Comment:
:pencil2: might want to rename these to be prepended with userPrefs since there are multiple loading states now

Reply:
Renamed with userPrefs prepended